Assets/GameAssets/Boss/BossScripts/Attacks/BaseBossAttack.cs

```diff
@@ -48,6 +53,40 @@ protected virtual void BeginAttack()
         _isAttackActive = true;
     }
 
+    protected virtual void PlayIdleLoop(float second)
+    {
+        if (_idleSFXCoroutine == null)
+        _idleSFXCoroutine = StartCoroutine(LoopIdleCorrectfloat(second));
+    }
+    protected virtual void DestroyIdleLoop()
+    {
+        if (_idleSFXCoroutine != null)
+        {
+            StopCoroutine(_idleSFXCoroutine);
+        }
+    }
+    /// <summary>
+    /// Play heartbeat at controlled rate 
+    /// </summary>
```

Comment:
XML comment is missing the float variable

Reply:
got it now